src/datastore/postgres/PgDatastore.ts

```diff
@@ -296,11 +296,19 @@ export class PgDatastore implements Datastore {
         date = date || new Date();
         const userId = (user instanceof SlackGhost) ? user.toEntry().id : user.userId;
 
-        await this.postgresDb.none("INSERT INTO metrics_activities (user_id, room_id, date) VALUES(${userId}, ${roomId}, ${date})", {
-            date: `${date.getFullYear()}-${date.getMonth()}-${date.getDate()}`,
-            roomId: room.toEntry().id,
-            userId,
-        });
+        try {
+            await this.postgresDb.none("INSERT INTO metrics_activities (user_id, room_id, date) VALUES(${userId}, ${roomId}, ${date})", {
```

Comment:
Hmm, have we considered fixing the statement to not conflict. e.g:

```sql
INSERT INTO distributors (did, dname) VALUES (7, 'Redline GmbH')
    ON CONFLICT cons_activities_unique DO NOTHING;
```

Reply:
I should brush up on my SQL. Thanks!

Reply:
```ON CONFLICT ON CONSTRAINT cons_activities_unique DO NOTHING;```